File: src/main/java/net/hectus/neobb/turn/person_game/block/PTCherryPressurePlate.java
package net.hectus.neobb.turn.person_game.block;

import net.hectus.neobb.player.NeoPlayer;
import net.hectus.neobb.turn.default_game.block.BlockTurn;
import net.hectus.neobb.turn.person_game.categorization.CounterCategory;
import org.bukkit.block.Block;

public class PTCherryPressurePlate extends BlockTurn implements CounterCategory {
    public PTCherryPressurePlate(NeoPlayer player) { super(player); }
    public PTCherryPressurePlate(Block data, NeoPlayer player) { super(data, player); }

    @Override
    public void apply() {
        super.apply();
        player.addArmor(1);
    }
}

File: src/main/kotlin/net/hectus/neobb/modes/turn/person_game/block/PTCherryPressurePlate.kt
package net.hectus.neobb.modes.turn.person_game.block

import com.marcpg.libpg.storing.Cord
import net.hectus.neobb.modes.turn.default_game.block.BlockTurn
import net.hectus.neobb.modes.turn.person_game.categorization.CounterCategory
import net.hectus.neobb.player.NeoPlayer
import org.bukkit.block.Block

class PTCherryPressurePlate(data: Block?, cord: Cord?, player: NeoPlayer?) : BlockTurn(data, cord, player), CounterCategory {
    override fun apply() {
        player!!.addArmor(1.0)
    }
}
